### ISSUE TITLE
Please fix chars map

### ISSUE THREAD

**Issue by zinzinday** [simov/slugify#49]
```json
[
   {
    "key": "Đ",
    "value": "D"
  },
  {
    "key": "đ",
    "value": "d"
  },
]
```

**Comment by simov** [comment]
Which language is this? The current translation to `dj` was added for Serbian.

**Comment by simov** [comment]
Ok, I saw your commits [here](https://github.com/zinzinday/slugify/commits/master), it's Vietnamese.

Now, that will be a hard problem to solve :thinking:

**Comment by simov** [comment]
Fixed in v1.4.3, you have to use the `locale` option instead. Check out the main readme.
